Box.V2/Models/BoxUser.cs

```diff
@@ -88,7 +89,7 @@ public class BoxUser : BoxEntity
         /// An array of key/value pairs set by the user’s admin
         /// </summary>
         [JsonProperty(PropertyName = FieldTrackingCodes)]
-        public string[] TrackingCodes { get; private set; }
```

Comment:
This might be a breaking change, would love to chat and grab more context before we merge, thanks!